src/plugins/discover/public/application/main/services/discover_app_state_container.ts

```diff
@@ -170,6 +176,13 @@ export const getDiscoverAppStateContainer = ({
     return !isEqualState(initialState, appStateContainer.getState());
   };
 
+  const getAppStateFromSavedSearch = (newSavedSearch: SavedSearch) => {
```

Comment:
A Jest test for this addition would be good.

Reply:
@davismcphee 

Thanks for catching this. I have added the tests.